<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgCollections</class>
 <widget class="QDialog" name="CQDlgCollections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Collections</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>411</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>Collection list</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqVisualizeCollection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Visualize selected collection</string>
    </property>
   </widget>
   <widget class="QListWidget" name="qqCollectionList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>391</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Operation on selected collection</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>371</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>What</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QRadioButton" name="qqAllObjects">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>171</width>
        <height>18</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>All objects</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="qqLooseObjects">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>171</width>
        <height>18</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>Selected objects</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="qqTrees">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>65</y>
        <width>201</width>
        <height>18</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>Tree of selected object</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="qqBaseIncluded">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>85</y>
        <width>171</width>
        <height>18</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>Base included</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="qqTipIncluded">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>125</y>
        <width>181</width>
        <height>18</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>Tip included</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="qqChains">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>105</y>
        <width>221</width>
        <height>18</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>Chain of selected object</string>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QToolButton" name="qqAdd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>181</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QToolButton" name="qqSubtract">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>181</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Subtract</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>391</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Composing elements and attributes</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="qqOverride">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Collection overrides collidable, measurable, renderable,
cuttable &amp;&amp; detectable properties</string>
     </property>
    </widget>
    <widget class="QListWidget" name="qqElementList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>371</width>
       <height>111</height>
      </rect>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolButton" name="qqNewCollection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Add new collection</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqNewCollection</tabstop>
  <tabstop>qqVisualizeCollection</tabstop>
  <tabstop>qqOverride</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
